frontend/src/configs/dev.js

```diff
@@ -1,5 +1,6 @@
 const configs = {
   API_URL: "https://api.cellxgene.dev.single-cell.czi.technology",
+  SENTRY_DEPLOYMENT_ENVIRONMENT: "development",
```

Comment:
setting up Sentry to capture CSP violation. However, this is not set up to capture build info like Explorer does at the moment